### PR TITLE
[1LP][RFR] New Test: Checking relationship value with trailing spaces

### PR DESCRIPTION
**Pull request by ganeshhubale**
- This test cases checks relationship value of instance with trailing spaces.
- Removed test ```test_quota_source``` because it is already automated in quota FA

{{ pytest: cfme/tests/automate/test_instance.py -k 'test_automate_relationship_trailing_spaces' -vv }}

### CODE REVIEW

File: cfme/tests/automate/test_instance.py [hunk 1]
```diff
@@ -113,3 +113,109 @@ def test_automate_instance_missing(domain, klass, namespace, appliance):
...
+    """
+    Handle trailing whitespaces in automate instance relationships.
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/10h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.6
```
Comment (sshveta) [inline cfme/tests/automate/test_instance.py]: Change this value to 5.9 or 5.10

File: cfme/tests/automate/test_instance.py [hunk 2]
```diff
@@ -113,3 +113,109 @@ def test_automate_instance_missing(domain, klass, namespace, appliance):
...
+               that has a relationship field.
+            2. Create an instance with the relationship field pointing to the first class'
+               instance but add a couple of whitespaces after it.
+            3. Execute the AE model, eg. using Simulate.
+        expectedResults:
+            1.
+            2.
+            3. Logs contain no resolution errors.
+
+    PR:
```
Comment (sshveta) [inline cfme/tests/automate/test_instance.py]: No need to mention PR . mention bugzilla instead if you have.

Comment (ganeshhubale) [reply cfme/tests/automate/test_instance.py]: There is no BZ for this fix. I found this PR only. So keeping as it is.

File: cfme/tests/automate/test_instance.py [hunk 3]
```diff
@@ -113,3 +113,109 @@ def test_automate_instance_missing(domain, klass, namespace, appliance):
...
+            "namespace": "{}/{}".format(domain.name, namespace.name),
+            "class": klass2.name,
+            "instance": instance2.name,
+        },
+    )
+
+    # Checking if automation log is giving resolution error or not by searching 'E,'.
+    # Also checking if method1 of klass1 is executed successfully or not by searching 'catch_string'
+    # in automation log.
+    for search in ['E,', catch_string]:
```
Comment (mshriver) [inline cfme/tests/automate/test_instance.py]: TODO: use LogValidator here instead of this.

**Pull request merged**